src/test/bench/shootout-reverse-complement.rs

```diff
@@ -155,7 +155,7 @@ impl<'a> Iterator for MutDnaSeqs<'a> {
     fn next(&mut self) -> Option<&'a mut [u8]> {
         let tmp = std::mem::replace(&mut self.s, &mut []);
         let tmp = match memchr(tmp, b'\n') {
-            Some(i) => tmp.slice_from_mut(i + 1),
+            Some(i) => &mut tmp[i + 1..],
```

Comment:
yes, and yes. I think some people tend to write `i+1..` (no spaces), to improve readability, but I think we don't have a convention for this (yet).